pyblish_lite/model.py

```diff
@@ -629,16 +630,17 @@ def filterAcceptsRow(self, source_row, source_parent):
         """Exclude items in `self.excludes`"""
         model = self.sourceModel()
         item = model.items[source_row]
-
         key = getattr(item, "filter", None)
         if key is not None:
             regex = self.filterRegExp()
             if regex.pattern():
                 match = regex.indexIn(key)
                 return False if match == -1 else True
-
         for role, values in self.includes.items():
             data = getattr(item, role, None)
+            if hasattr(data, '__iter__'):
+                return True if any(value in values for value in data) else False
```

Comment:
Thanks @p4vv37! Don't worry about reproducible, I'll put something together for the release this time. For screenshot, if you find some way of taking one or two to illustrate the feature, you can drag/drop them into the comment field of this thread to upload it. Then I can include this in the release notes.

Great job and Merry Christmas!